Extensions/TopDownMovementBehavior/topdownmovementruntimebehavior.ts

```diff
@@ -322,6 +375,40 @@ namespace gdjs {
       this._downKey = true;
     }
   }
+
+  export interface BasisTransformation {
+    toScreen(worldPoint: FloatPoint, screenPoint: FloatPoint): void;
+  }
+
+  export class IsometryTransformation implements BasisTransformation {
+    screen: float[][];
```

Comment:
nitpicking: put a `_` as the first character to indicate that this should not be used outside of the class.